ui/src/main/ui/src/triggers/triggers-service.js

```diff
@@ -33,5 +31,33 @@ angular.module('hwk.triggersModule').service('hwk.triggersService', ['$resource'
         }
       });
     };
+
+    this.NewTrigger = function (tenantId) {
```

Comment:
This is just a nitpick but why the capital letters on method names in services? When elsewhere it seems like lowercase (i.e. `$scope.newTrigger`)